spec/helpers/capybara_poltergeist_screenshot.rb

```diff
@@ -0,0 +1,12 @@
+require 'launchy'
+
+module Capybara
+  module PoltergeistScreenshot
+    def screenshot_and_open_image(full: false)
```

Comment:
I was asking because I think `save_and_open_screenshot` was working for me in javascript tests without any additional gems.